### ISSUE TITLE
python3-pyinfra: dependencies are broken

### ISSUE THREAD

**Issue by hazayan** [void-linux/void-packages#27885]
### System

* xuname:  
  **Void 5.9.16_1 x86_64 GenuineIntel uptodate rFFFF**
* package:  
  *affected package(s) including the version*: **python3-pyinfra-1.3.1_1**

### Expected behavior

Invoking pyinfra on the command line would run without dependency errors

### Actual behavior

Running the command below result in  the subsequent trace:

```
.... truncated for brevity ....

File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3222, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3251, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 567, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 884, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'distro==1.5.0' distribution was not found and is required by pyinfra
```

Upon fixing this by manually installing `python3-distro`, running the cli results in the subsequent trace:

```
.... truncated for brevity ....

File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3222, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3251, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 569, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 582, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python3.9/site-packages/pkg_resources/__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'gevent>=1.5' distribution was not found and is required by pyinfra
```

`python3-gevent` version currently is `1.4.0`. Working around that by locally updating the package template to the latest upstream version, then building and installing it, fixes the issue.

### Steps to reproduce the behavior

```
$ pyinfra @local exec -- echo "hello world"
```

**Comment by UsernameRandomlyGenerated** [comment]
https://github.com/void-linux/void-packages/pull/27040 would fix that I guess

**Comment by leahneukirchen** [comment]
Fixed in 751648b779.
